GithubApp/Models/Person.swift

```diff
@@ -12,14 +12,24 @@ struct Person: Decodable {
     var name: String?
     var owner: Owner?
     var description: String?
-    var languageUrl: String
+    var languageUrl: languages = {
+        //        let languagesData = ["Python","Interior Painting","Electrician","Inappropriate client behavior","Gardener","Carpenter","House Cleaning"]
+        let randomInt = Int.random(in: 0...6)
+        return randomInt == 0 ? .Python :
+        randomInt == 1 ? .InteriorPainting :
+        randomInt == 2 ? .Electrician :
+        randomInt == 3 ? .InappropriateClientBehavior :
+        randomInt == 4 ? .Gardener :
+        randomInt == 5 ? .Carpenter :
+        randomInt == 6 ? .HouseCleaning : .Python
```

Comment:
improve this logic.

Reply:
changed with
let languageData = [Languages.python, Languages.interiorPainting, Languages.electrician, Languages.inappropriateClientBehavior, Languages.gardener,Languages.carpenter,Languages.houseCleaning]
        let randomInt = Int.random(in: 0...6)
        return languageData[randomInt]